website_generator.py

```diff
@@ -0,0 +1,43 @@
+import flask
+from flask_flatpages import FlatPages
+from flask_frozen import Freezer
+from flask import Flask, request, send_from_directory, render_template
+from shutil import copyfile
+import os
+import shutil
+import numpy
+from market_scanner import mainObj
+
+# this is used by me to generate the web page you can find at: https://sampom100.github.io/UnusualVolumeDetector/
+
+app = flask.Flask(__name__, static_url_path='')
+app.config["DEBUG"] = False
+app.config['SECRET_KEY'] = 'deditaded wam'
```

Comment:
There's got to be some way to not have your secrets in plain text on a public repository

Reply:
haha yeah, I pretty much completely ripped the flask script from another project and noticed this

maybe i should add more deditated wam to the server